.babelrc

```diff
@@ -1,13 +1,21 @@
 {
   "presets": [
-    ["@babel/env", {
-      "targets": {"node": "10"}
-    }],
+    [
+      "@babel/env", {
+        "targets": {"node": "14"},
```

Comment:
I am wondering if this is needed given we should use the right node version in the CI job that does the build + publish.

I am actually wondering if that wouldn't actually have the opposite effect and hide potential bugs on Node > 14 (because that will force babel to stick to Node 14 features).